lightkurve/seismology/core.py

```diff
@@ -276,11 +353,170 @@ def plot_echelle(self, deltanu=None, numax=None,
+        return fig, stretch_slider
+
+
+    def interact_echelle(self, notebook_url='localhost:8888', **kwargs):
+        """ Shows an interactive Echelle diagram based on a Seismology object.
+        """
+
+        try:
+            import bokeh
+            if bokeh.__version__[0] == '0':
+                warnings.warn("interact() requires Bokeh version 1.0 or later", LightkurveWarning)
+        except ImportError:
+            log.error("The interact() tool requires the `bokeh` Python package; "
+                      "you can install bokeh using e.g. `conda install bokeh`.")
+            return None
+
+        maximum_frequency = kwargs.pop('maximum_frequency', self.periodogram.frequency.max().value)
+        minimum_frequency = kwargs.pop('minimum_frequency', self.periodogram.frequency. min().value)
+
+        if not hasattr(self, 'deltanu'):
+            dnu = SeismologyQuantity(quantity=self.periodogram.frequency.max()/30,
+                                                       name='deltanu', method='echelle')
+        else:
+            dnu = self.deltanu
+
+        if (not hasattr(dnu, 'unit')):
+            dnu = dnu * self.periodogram.frequency.unit
+
+        def create_interact_ui(doc):
+            fig_tpf, stretch_slider = self._make_echelle_elements(dnu,
+                                                maximum_frequency=maximum_frequency,
+                                                minimum_frequency=minimum_frequency,
+                                                **kwargs)
+            maxdnu = self.periodogram.frequency.max().value/5
+            # Interactive slider widgets
+            dnu_slider = Slider(start=0.01,
+                                end=maxdnu,
+                                value=dnu.value,
+                                step=0.001,
+                                title="Delta Nu",
+                                width=290)
+            r_button = Button(label=">", button_type="default", width=30)
+            l_button = Button(label="<", button_type="default", width=30)
+            rr_button = Button(label=">>", button_type="default", width=30)
+            ll_button = Button(label="<<", button_type="default", width=30)
+
+
+            def update(attr, old, new):
+                """Callback to take action when dnu slider changes"""
+                dnu = SeismologyQuantity(quantity=dnu_slider.value*u.microhertz,
+                                                       name='deltanu', method='echelle')
+                ep, x_f, y_f = self._clean_echelle(deltanu=dnu,
+                                                    minimum_frequency=minimum_frequency,
+                                                    maximum_frequency=maximum_frequency,
+                                                    **kwargs)
+                fig_tpf.select('img')[0].data_source.data['image'] = [ep.value]
+                fig_tpf.xaxis.axis_label = r'Frequency / {:.3f} Mod. 1'.format(dnu)
```

Comment:
We should use the same unit axis labels as we use in `periodogram`, and the same phrasing as in `plot_echelle()` e.g.: 

```
axis_label = rf" Frequency mod. {dnu.value:.3f} [{deltanu.unit.to_string('latex')}]"
```

Reply:
Hi Ollie, the x axis is actually different in the interact echelle diagram. In order for the slider to be "smooth" the x axis is 

`frequency/dnu %  1`

Which means the x axis spans 0 to 1 in phase

In the original echelle diagram the x axis is

`frequency %  dnu`

So it goes from 0 to dnu.